### PR TITLE
[new] add bootstrap-timepicker@0.5.2

### PR DESCRIPTION
**Pull request by Amomo**
Hi @maruilian11 this PR is for #8155

I add `bootstrap-timepicker` at `v0.5.2` and add `npm` auto-update config in its package.json.

Would you mind checking this PR for me? Thank you~ :-D

# Profile of the lib
 * Git repository (required): https://github.com/jdewit/bootstrap-timepicker
 * Official website (optional, not the repository): https://jdewit.github.io/bootstrap-timepicker/
 * NPM package url (optional): https://www.npmjs.com/package/bootstrap-timepicker
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 110
   - Count of watchers: 1438
   - Count of forks: 1050
 * NPM download stats (optional):
   - Downloads in the last day: 7
   - Downloads in the last week: 398
   - Downloads in the last month: 1668

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
please use `bootstrap-timepicker*` or `bootstrap-timepicker*.+(js|map)` in filemap
so as css file

**Comment by maruilian11** [comment]
miss keyword field

**Comment by Amomo** [comment]
@maruilian11 thanks for your review!
I have fix the filemap, add the lost .map file and add keyword field.
Could you help me checking this PR again? Thank you!

File: ajax/libs/bootstrap-timepicker/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "type": "git",
+    "url": "git://github.com/jdewit/bootstrap-timepicker.git"
+  },
+  "npmName": "bootstrap-timepicker",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/bootstrap-timepicker*",
+        "css/bootstrap-timepicker*"
```
Comment (maruilian11) [inline ajax/libs/bootstrap-timepicker/package.json]: you can use `+(js|css)/bootstrap-timepicker*` next time.

**Comment by maruilian11** [comment]
@Amomo if the author already provide min file, we don't have to add map file for it.
only when they provide only full js/css file, i make min file and map file for it.
you can do it next time.

@PeterDaveHello i think this PR is ok.

**Comment by PeterDaveHello** [comment]
@maruilian11 So what's the situation here? Some manually minified files with map files? I don't want to most non-official minified files instead of official ones.

**Comment by Amomo** [comment]
oh my bad! I didn't notice that the minified file is from official when I fixing this PR.
I shouldn't remove the official files. Sorry.

**Comment by Amomo** [comment]
@maruilian11 thanks for your review!
I have put the official minified files back, could you help me checking this PR again?
Thank you!

**Comment by maruilian11** [comment]
@PeterDaveHello i think this PR is ok.

**Comment by PeterDaveHello** [comment]
It's npm package only contains certain releases, please try to manually the other versions.

**Comment by PeterDaveHello** [comment]
Should mention the auto-update on title next time.

**Pull request merged**